src/Bundle/Command/SelfUpdateCommand.php

```diff
@@ -80,6 +86,8 @@ private function stable(Updater $updater)
      */
     public function isEnabled()
     {
+        return true;
```

Comment:
@wachterjohannes sure?

Reply:
@alexander-schranz nope 😂  but no problem in the compiled phar its always true - like this its only a problem in development - fixed it in https://github.com/nanbando/core/commit/998ff9f842a395bc16c58ec8aa62d09c3d276f5a